src/ontology/slurp/omim.tsv

```diff
@@ -2,25 +2,13 @@ mondo_id	mondo_label	xref	xref_source	original_label	definition	parents
 ID	LABEL	A oboInOwl:hasDbXref	>A oboInOwl:source SPLIT=|		A IAO:0000115	SC %
 MONDO:0958222	maple syrup urine disease,  iia 1b	OMIM:620698	MONDO:equivalentTo	maple syrup urine disease,  iia 1b		
 MONDO:0958223	maple syrup urine disease,  iia 2	OMIM:620699	MONDO:equivalentTo	maple syrup urine disease,  iia 2		
-MONDO:0958224	encephalopathy, porphyria-related	OMIM:620704	MONDO:equivalentTo	encephalopathy, porphyria-related		
-MONDO:0958225	epidermolytic hyperkeratosis 2b, autosomal recessive	OMIM:620707	MONDO:equivalentTo	epidermolytic hyperkeratosis 2b, autosomal recessive		
-MONDO:0958226	leukoencephalopathy, porphyria-related	OMIM:620711	MONDO:equivalentTo	leukoencephalopathy, porphyria-related		
+MONDO:0958225	epidermolytic hyperkeratosis 2b, autosomal recessive	OMIM:620707	MONDO:equivalentTo	epidermolytic hyperkeratosis 2b, autosomal recessive		MONDO:0957316
```

Comment:
## Updated slurp output
This isn't really necessary for this PR, and I can undo this change if requested, but I just re-ran `migrate.py` directly for OMIM without updating any prerequisites, etc, and confirmed the fix.